app/resources/en/translation/common.json

```diff
@@ -12,7 +12,7 @@
   "acceptAndContinue": "Accept and continue",
   "send": "Send",
   "ogl": "All content is available under the <a href=\"https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/\" rel=\"license\">Open Government Licence v3.0</a>, except where otherwise stated",
-  "cookieText": "GOV.UK uses cookies to make the site simpler. <a href=\"#\" title=\"Find out more about cookies\">Find out more about cookies</a>",
+  "cookieText": "GOV.UK uses cookies to make the site simpler. <a href=\"/cookies\" rel=\"noopener\" title=\"Find out more about cookies\">Find out more about cookies</a>",
```

Comment:
Ideally at some stage it would be good to reference the `/cookies` url from the config as we have with other urls.

Also check if we need to use `noopener` on internal urls.